packages/split-button/docs/SplitButton.stories.tsx

```diff
@@ -0,0 +1,207 @@
+      options: ["Button", "Anchor"],
+      control: {
+        type: "select",
+        labels: {
+          Button: '{ label: "Edit Survey", onClick: action("clicked") }',
+          Anchor: '{ label: "Edit Survey", href: "//example.com" }',
+        },
+      },
+      mapping: {
+        Button: ACTION_BUTTON_PROPS__BUTTON,
+        Anchor: ACTION_BUTTON_PROPS__ANCHOR,
+      },
+    },
+    dropdownContent: {
+      options: [
+        "MenuList - MenuItems enabled",
+        "MenuList - one MenuItem disabled",
+      ],
+      control: { type: "select" },
+      mapping: {
+        "MenuList - MenuItems enabled": DROPDOWN_CONTENT__ENABLED,
+        "MenuList - one MenuItem disabled": DROPDOWN_CONTENT_ONE__DISABLED,
+      },
+    },
+  },
+  parameters: {
+    docs: {
+      description: {
+        component: '`import { SplitButton } from "@kaizen/split-button"`',
+      },
+    },
+    ...figmaEmbed(
+      "https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%E2%9D%A4%EF%B8%8F-UI-Kit%3A-Heart?node-id=14512%3A404"
+    ),
+  },
+  decorators: [withDesign],
+} as ComponentMeta<typeof SplitButton>
+
+export const DefaultKaizenSiteDemo = args => <SplitButton {...args} />
+DefaultKaizenSiteDemo.storyName = "Split Button"
+DefaultKaizenSiteDemo.args = {
+  actionButtonProps: "Button",
+  dropdownContent: "MenuList - MenuItems enabled",
+}
+
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => {
+  const ROWS_MAP: Array<{ rowTitle: string } & SplitButtonProps> = [
+    {
+      rowTitle: "Default",
+      actionButtonProps: ACTION_BUTTON_PROPS__BUTTON,
+      dropdownContent: DROPDOWN_CONTENT__ENABLED,
+    },
+    {
+      rowTitle: "RTL",
+      actionButtonProps: ACTION_BUTTON_PROPS__BUTTON,
+      dropdownContent: DROPDOWN_CONTENT__ENABLED,
+      dir: "rtl",
+    },
+    {
+      rowTitle: "Anchor Link",
+      actionButtonProps: ACTION_BUTTON_PROPS__ANCHOR,
+      dropdownContent: DROPDOWN_CONTENT__ENABLED,
+    },
+    {
+      rowTitle: "With Disabled Dropdown Option",
+      actionButtonProps: ACTION_BUTTON_PROPS__BUTTON,
+      dropdownContent: DROPDOWN_CONTENT_ONE__DISABLED,
+    },
+  ]
+
+  return (
+    <>
+      <StoryWrapper isReversed={isReversed}>
+        <StoryWrapper.RowHeader headings={["Base", "Disabled"]} />
+        {ROWS_MAP.map(({ rowTitle, ...props }) => (
+          <StoryWrapper.Row key={rowTitle} rowTitle={rowTitle}>
+            <SplitButton isReversed={isReversed} {...props} />
+            <SplitButton isReversed={isReversed} disabled {...props} />
+          </StoryWrapper.Row>
+        ))}
+      </StoryWrapper>
+
+      <StoryWrapper isReversed={isReversed}>
+        <StoryWrapper.RowHeader headings={["Hover", "Focus"]} />
+        <StoryWrapper.Row rowTitle="Action button">
+          <SplitButton
+            isReversed={isReversed}
+            actionButtonProps={{
+              ...ACTION_BUTTON_PROPS__BUTTON,
+              classNameOverride: "__hover",
```

Comment:
Just wary on this since we don't usually showcase hover or focus states - do think this is something we should start doing? or is it more relevant here because split button has two focusable elements

Reply:
We should actually be doing this more often so we can have Chromatic help us pick up if there are changes between the snapshots :)